Add Hindi translation to FES

Ayush23Dash opened the pull request:
<!--
  Thank you for contributing! Please use this pull request (PR) template.


 In the description field of this PR, include "resolves #XXXX" tagging the issue you are fixing. If this PR addresses the issue but doesn't completely resolve it (ie the issue should remain open after your PR is merged), write "addresses #XXXX".-->


 Changes:
<!-- Add here what changes were made in this pull request and if possible provide links showcasing the changes. -->
Add Hindi(hi) translation to FES

#### PR Checklist
<!--
  To check any option, replace the "[ ]" with a "[x]". Be sure to check out how it looks in the Preview tab! Feel free to remove any portion of the template that is not relevant for your issue.
-->

- [x] `npm run lint` passes
- [x] [Inline documentation] is included / updated
- [x] [Unit tests] are included / updated

[Inline documentation]: https://github.com/processing/p5.js/blob/main/contributor_docs/inline_documentation.md
[Unit tests]: https://github.com/processing/p5.js/tree/main/contributor_docs#unit-tests

Ayush23Dash commented:
@almchung @nbriz could you please review it

Qianqianye commented:
Thanks @Ayush23Dash! 

I'm also inviting the [Hindi translation stewards](https://github.com/processing/p5.js-website/issues/1220) @adarrssh, @Divyansh013, @deepchauhan, @SarveshLimaye, @ranaaditya, and @SanjaySinghRajpoot to review the translation.

ranaaditya commented on translations/hi/translation.json:
```diff
@@ -0,0 +1,73 @@
+        "syntax": {
+          "badReturnOrYield": "\nसिंटैक्स त्रुटि - रिटर्न किसी फ़ंक्शन के बाहर होता है। सुनिश्चित करें कि आपसे कोई कोष्ठक नहीं छूट रहा है, ताकि रिटर्न किसी फ़ंक्शन के अंदर हो।\n\n+ अधिक जानकारी: {{url}}",
+          "invalidToken": "\nसिंटैक्स त्रुटि - एक ऐसा प्रतीक मिला जिसे जावास्क्रिप्ट नहीं पहचानता या अपने स्थान पर अपेक्षित नहीं था।\n\n+ अधिक जानकारी: {{url}}",
+          "missingInitializer": "\nसिंटैक्स त्रुटि - एक कॉन्स्ट वैरिएबल घोषित किया गया है लेकिन प्रारंभ नहीं किया गया है। जावास्क्रिप्ट में, एक कॉन्स्ट के लिए इनिशियलाइज़र की आवश्यकता होती है। एक मान उसी स्टेटमेंट में निर्दिष्ट किया जाना चाहिए जिसमें वेरिएबल घोषित किया गया है। त्रुटि में पंक्ति संख्या की जाँच करें और स्थिरांक चर को एक मान निर्दिष्ट करें।\n\n+ अधिक जानकारी: {{url}}",
```
just a suggestion here- Translation is not only about converting English to Hindi. Its about translating and conversion and translation has some difference.

Ex- for *variable, initializer, etc* many words- you just write it in hindi instead of converting it into hindi.

Please find these mistakes in your PR and let me know when those are fixed. I have seen many such mistakes in this file.

Thanks,
Aditya Rana

ranaaditya commented:
> Thanks @Ayush23Dash!
> 
> I'm also inviting the [Hindi translation stewards](https://github.com/processing/p5.js-website/issues/1220) @adarrssh, @Divyansh013, @deepchauhan, @SarveshLimaye, @ranaaditya, and @SanjaySinghRajpoot to review the translation.

hi @Qianqianye , reviewed this PR, suggested some changes.

Thanks,
Aditya Rana

Ayush23Dash commented:
Hi @ranaaditya 
Thanks for reviewing the PR.
I have updated the hindi translations with the following exceptions.
I did not change the following words as this might change the literal meaning of the sentence and all of the below mentioned words are used as it is in Hindi in Coding Terminology : 

Words that I am keeping as it is(Just converted to Hindi) are:
1. Media(मीडिया)
2. Browser(ब्राउज़र)
3.File(फ़ाइल)
4. File Load(फ़ाइल लोड)
5. Online(ऑनलाइन)
6. Host(होस्ट)
7.Server(सर्वर)
8. Return(रिटर्न)
9. Function(फ़ंक्शन)
10. Javascript(जावास्क्रिप्ट)
11. Library(लाइब्रेरी)
12. Preload(प्रीलोड)

I have changed other words to their Hindi meanings!

ranaaditya commented:
> Hi @ranaaditya Thanks for reviewing the PR. I have updated the hindi translations with the following exceptions. I did not change the following words as this might change the literal meaning of the sentence and all of the below mentioned words are used as it is in Hindi in Coding Terminology :
> 
> Words that I am keeping as it is(Just converted to Hindi) are:
> 
> 1. Media(मीडिया)
> 2. Browser(ब्राउज़र)
>    3.File(फ़ाइल)
> 3. File Load(फ़ाइल लोड)
> 4. Online(ऑनलाइन)
> 5. Host(होस्ट)
>    7.Server(सर्वर)
> 6. Return(रिटर्न)
> 7. Function(फ़ंक्शन)
> 8. Javascript(जावास्क्रिप्ट)
> 9. Library(लाइब्रेरी)
> 10. Preload(प्रीलोड)
> 
> I have changed other words to their Hindi meanings!

Hi @Ayush23Dash , yeah that make sense in this case.

Cool will review the PR and update.

Thanks !

ranaaditya commented:
Hi @Qianqianye this PR LGTM now after changes.

Thanks,
Aditya Rana

limzykenneth commented:
Looks good. Thanks!
